rest/resource-server/pom.xml

```diff
@@ -204,6 +204,16 @@
             <artifactId>springdoc-openapi-starter-common</artifactId>
             <version>${springdoc-openapi-stater-common.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.spdx</groupId>
+            <artifactId>tools-java</artifactId>
+            <version>${org.spdx.tools.java.version}</version>
```

Comment:
The version field can be ignored here as it is part of parent `pom.xml` already.